### PR TITLE
Use content_tag_for and div_for in views

### PR DESCRIPTION
**Pull request by allenwq**

### CODE REVIEW

**Comment by lowjoel** [comment]
Is that all? All the other index and show actions?

**Comment by allenwq** [comment]
Announcement index is not quite suitable though, nvm, I'll update the code and later you see again.

**Comment by lowjoel** [comment]
What do you mean?

**Comment by allenwq** [comment]
@lowjoel  Updated here

File: app/views/admin/announcements/_announcement.html.slim [hunk 1]
```diff
@@ -1,4 +1,4 @@
-div.announcement class=time_period_class(announcement)
+= div_for(announcement, class: ['announcement', time_period_class(announcement)])
```
Comment (lowjoel) [inline app/views/admin/announcements/_announcement.html.slim]: div_for adds the type of the model as a class already.

Comment (allenwq) [inline app/views/admin/announcements/_announcement.html.slim]: If you specify the class, it will not add and just use the class you have specified.

Comment (lowjoel) [inline app/views/admin/announcements/_announcement.html.slim]: Look at the example in the Rails documentation. It adds the class, unless you're saying the documentation is wrong of which then you should submit a PR to have that fixed :)

Comment (allenwq) [inline app/views/admin/announcements/_announcement.html.slim]: Wait let me check

Comment (allenwq) [inline app/views/admin/announcements/_announcement.html.slim]: Oh, You are right ! few tests failed, so I think the class it's not added.

**Pull request merged**